tools/roslaunch/scripts/roscore

```diff
@@ -49,6 +50,12 @@ def _get_optparse():
     parser.add_option("-v", action="store_true",
                       dest="verbose", default=False,
                       help="verbose printing")
+    parser.add_option("-w", "--numworkers",
+                      dest="num_workers", default=NUM_WORKERS, type="int",
```

Comment:
While grepping, I found this same mistake in [rosmsg](https://github.com/ros/ros_comm/blob/indigo-devel/tools/rosmsg/src/rosmsg/__init__.py#L262) and [rosnode](https://github.com/ros/ros_comm/blob/indigo-devel/tools/rosnode/src/rosnode/__init__.py#L726).